repos/fdbt-site/src/pages/viewCaps.tsx

```diff
@@ -25,20 +35,82 @@ const ViewCap = ({ viewCapErrors = [] }: CapProps): ReactElement => {
                 <div className="govuk-grid-column-three-quarters">
                     <h1 className="govuk-heading-xl">Caps</h1>
                     <p className="govuk-body govuk-!-margin-bottom-8">
-                        Define your different types of caps and their expiries
+                        Define your different types of caps and when they expire
                     </p>
+
+                    <div>
+                        <h2 className="govuk-heading-l">Cap expiry</h2>
+                        {capValidity ? (
+                            <CapExpiryCard capValidity={capValidity} fareDayEnd={fareDayEnd} />
+                        ) : (
+                            <>
+                                <p className="govuk-body">
+                                    <em>You currently have not set up cap expiry.</em>
+                                </p>
+                                <a className="govuk-button" data-module="govuk-button" href="/selectCapValidity">
+                                    Add cap expiry
+                                </a>
+                            </>
+                        )}
+                    </div>
                 </div>
             </div>
         </BaseLayout>
     );
 };
 
-export const getServerSideProps = (): { props: CapProps } => {
+const CapExpiryCard = ({ capValidity, fareDayEnd }: CapExpiryCardProps): ReactElement => {
+    return (
+        <>
+            <div className="card-row">
+                <div className="card" id="cap-expiry">
+                    <div className="card__body">
+                        <div className="card__actions">
+                            <ul className="actions__list">
+                                <li className="actions__item">
+                                    <a
+                                        className="govuk-link govuk-!-font-size-16 govuk-!-font-weight-regular"
+                                        href="/selectCapValidity"
+                                    >
+                                        Edit
+                                    </a>
+                                </li>
+                            </ul>
+                        </div>
+
+                        <h4 className="govuk-heading-m govuk-!-padding-bottom-4">
+                            {capValidity === 'endOfCalendarDay'
+                                ? 'At the end of a calendar day'
```

Comment:
Could we make this text a little bit smaller, and include the text from the selectCapValidity page here? For example, if they select 'At the end of a calendar day', it would show

At the end of a calendar day (big text)

The cap applies to journeys made before midnight (smaller text, grey colour).